File: Frontend/news/src/app/core/Enums/site-file-type.ts
export enum SiteFileType {
  Rules = 0,
  HomeTopImage = 1,
  HomeTopImageMobile = 2,
  // FirstAdImage = 3,
  // SecondAdImage = 4,
  FirstLinkImage = 5,
  SecondLinkImage = 6,
  ThirdLinkImage = 7,
  ForthLinkImage = 8,
  FirstSocialLinkImage = 9,
  SecondSocialLinkImage = 10,
  ThirdSocialLinkImage = 11,
  ForthSocialLinkImage = 12,
}

export const SiteFileDisplayNames: {
  [key in keyof typeof SiteFileType]?: string;
} = {
  Rules: 'قوانین و مقررات',
  HomeTopImage: 'تصویر بالای صفحه (دسکتاپ)',
  HomeTopImageMobile: 'تصویر بالای صفحه (موبایل)',
  // FirstAdImage: 'بنر اول',
  // SecondAdImage: 'بنر دوم',
  FirstLinkImage: 'عکس لینک اول',
  SecondLinkImage: 'عکس لینک دوم',
  ThirdLinkImage: 'عکس لینک سوم',
  ForthLinkImage: 'عکس لینک چهارم',
  FirstSocialLinkImage: 'عکس فضای مجازی اول',
  SecondSocialLinkImage: 'عکس فضای مجازی دوم',
  ThirdSocialLinkImage: 'عکس فضای مجازی سوم',
  ForthSocialLinkImage: 'عکس فضای مجازی چهارم',
};

File: Backend/news/news/01-Domain/Enum/SiteFileType.cs
using System.ComponentModel.DataAnnotations;

namespace news._01_Domain.Enum
{
    public enum SiteFileType:byte
    {
        [Display(Name ="قوانین و مقررات")] Rules=0,
        [Display(Name ="عکس بنر بالای صفحه اصلی")] HomeTopImage = 1,
        [Display(Name ="عکس بنر بالای صفحه اصلی (موبایل) ")] HomeTopImageMobile=2,
        [Display(Name ="عکس تبلیغات 1")] FirstAdImage=3,
        [Display(Name ="عکس تبلیغات 2")] SecondAdImage = 4,
        [Display(Name ="عکس لینک 1")] FirstLinkImage = 5,
        [Display(Name ="عکس لینک 2")] SecondLinkImage = 6,
        [Display(Name ="عکس لینک 3")] ThirdLinkImage = 7,
        [Display(Name ="عکس لینک 4")] ForthLinkImage = 8,
        [Display(Name = "عکس لینک فضای مجازی 1")] FirstSocialLinkImage = 9,
        [Display(Name = "عکس لینک فضای مجازی 2")] SecondSocialLinkImage = 10,
        [Display(Name = "عکس لینک فضای مجازی 3")] ThirdSocialLinkImage = 11,
        [Display(Name = "عکس لینک فضای مجازی 4")] ForthSocialLinkImage = 12,
    }
}
